prod-cockpit-rcar.yaml

```diff
@@ -5,6 +5,7 @@ variables:
   YOCTOS_WORK_DIR: "yocto"
   DOM0_BUILD_DIR: "build-dom0"
   DOMD_BUILD_DIR: "build-domd"
+  CLUSTER_BUILD_DIR: "cluster-wrapper-build"
```

Comment:
Please rename the folder to `build-cluster-wrapper` so it aligns with build folders for other components, like `build-domd`.